source/gui/settingsDialogs.py

```diff
@@ -2693,6 +2698,397 @@ def onSave(self):
 		)
 
 
+class SpeechOptions(Enum):
+	DecimalSeparator = ("Auto", ".", ",", "Custom")
+	Impairment = ("LearningDisability", "Blindness", "LowVision")
+	Verbosity = ("Terse", "Medium", "Verbose")
+	SubjectArea = ("General",)
+	Chemistry = ("SpellOut", "Off")
+
+
+class NavigationOptions(Enum):
+	NavMode = ("Enhanced", "Simple", "Character")
+	NavVerbosity = ("Terse", "Medium", "Verbose")
+	CopyAs = ("MathML", "LaTeX", "ASCIIMath", "Speech")
+
+
+class BrailleOptions(Enum):
+	BrailleNavHighlight = ("Off", "FirstChar", "EndPoints", "All")
+
+
+# two constants to scale "PauseFactor"
+# these work out so that a slider that goes [0,14] has value ~100 at 7 and ~1000 at 14
+PAUSE_FACTOR_SCALE: float = 9.5
+PAUSE_FACTOR_LOG_BASE: float = 1.4
+
+
+class MathSettingsPanel(SettingsPanel):
+	# Translators: Title of the math settings panel.
+	title = pgettext("math", "Math")
+	# Translators: The Help ID of the math settings panel.
+	helpId = pgettext("math", "MathSettings")
```

Comment:
This should not be translated

```suggestion
	helpId = "MathSettings"
```